README.md

```diff
@@ -157,43 +171,16 @@ include(${_project_options_SOURCE_DIR}/src/GlobalOptions.cmake)
 # Set the project name and language
 project(myproject LANGUAGES CXX)
 
-# ❗ enable sanitizers if running the tests
-option(FEATURE_TESTS "Enable the tests" OFF)
-if(FEATURE_TESTS)
-    if(CMAKE_SYSTEM_NAME STREQUAL "Windows")
-        set(ENABLE_SANITIZER_ADDRESS OFF)
-        set(ENABLE_SANITIZER_UNDEFINED_BEHAVIOR OFF)
-    else()
-        set(ENABLE_SANITIZER_ADDRESS "ENABLE_SANITIZER_ADDRESS")
-        set(ENABLE_SANITIZER_UNDEFINED_BEHAVIOR "ENABLE_SANITIZER_UNDEFINED_BEHAVIOR")
-    endif()
-endif()
+# Set PCH to be on by default for all non-Developer Mode Builds
+# (this is just intended as an example of what is possible)
+set(ENABLE_PCH_USER_DEFAULT ON)
 
 # Initialize project_options variable related to this project
 # This overwrites `project_options` and sets `project_warnings`
 # uncomment the options to enable them:
-project_options(
-      ENABLE_CACHE
-      ENABLE_CPPCHECK
-      ENABLE_CLANG_TIDY
-      # WARNINGS_AS_ERRORS
-      # ENABLE_CONAN
-      # ENABLE_IPO
-      # ENABLE_INCLUDE_WHAT_YOU_USE
-      # ENABLE_COVERAGE
-      # ENABLE_PCH
-      # PCH_HEADERS
-      # ENABLE_DOXYGEN
-      # ENABLE_IPO
-      # ENABLE_USER_LINKER
-      # ENABLE_BUILD_WITH_TIME_TRACE
-      # ENABLE_UNITY
-      # ❗ Now, the address and undefined behavior sanitizers are enabled through CMake options
-      ${ENABLE_SANITIZER_ADDRESS}
-      ${ENABLE_SANITIZER_UNDEFINED_BEHAVIOR}
-      # ENABLE_SANITIZER_LEAK
-      # ENABLE_SANITIZER_THREAD
-      # ENABLE_SANITIZER_MEMORY
+dynamic_project_options(
+  # set PCH headers you want enabled. Format can be slow, so this might be helpful
+  PCH_HEADERS <vector> <string> <fmt/format.h>
```

Comment:
Personally, I am not a fan of PCH headers especially now that modules support is being improved.  PCH headers usually fail on me. Is it really recommended to use them by default? 

```suggestion
  # PCH_HEADERS <vector> <string> <fmt/format.h>
```

Reply:
Above it just says "this is an example of what is possible" it is not the default. https://github.com/cpp-best-practices/project_options/pull/46/files/a7eb218f05a2ee9ef6a9d4074a67f1e76cb9f90f#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R175

But a few notes:

 * CMake's PCH support is extremely stable. I have not had a problem with it in the past couple of years
 * In the example I only set it as the default for User Build Mode, not Developer mode. The problem with it being set in developer mode is that it globally adds a set of header files and developers start to accidentally rely on that
 * The CI does test Developer and User build modes, so any accidental breakage will be caught